src/App/pages/LandingPage/Main/Carousel/Carousel.js

```diff
@@ -0,0 +1,128 @@
+                            <h3>Movie name</h3>
+                        </div>
+                    </div>
+                    <div className='carousel__item'>
+                        <div className='carousel__img-container'>
+                            <img src={poster_1} />
+                            <div className='carousel__ticket'>
+                                <a href="#"><Ticket /></a>
+                            </div>
+                            <h3>Movie name</h3>
+                        </div>
+                    </div>
+                    <div className='carousel__item'>
+                        <div className='carousel__img-container'>
+                            <img src={poster_2} />
+                            <div className='carousel__ticket'>
+                                <a href="#"><Ticket /></a>
+                            </div>
+                            <h3>Movie name</h3>
+                        </div>
+                    </div>
+                    <div className='carousel__item'>
+                        <div className='carousel__img-container'>
+                            <img src={poster_3} />
+                            <div className='carousel__ticket'>
+                                <a href="#"><Ticket /></a>
+                            </div>
+                            <h3>Movie name</h3>
+                        </div>
+                    </div>
+                    <div className='carousel__item'>
+                        <div className='carousel__img-container'>
+                            <img src={poster_1} />
+                            <div className='carousel__ticket'>
+                                <a href="#"><Ticket /></a>
+                            </div>
+                            <h3>Movie name</h3>
+                        </div>
+                    </div>
+                    <div className='carousel__item'>
+                        <div className='carousel__img-container'>
+                            <img src={poster_2} />
+                            <div className='carousel__ticket'>
+                                <a href="#"><Ticket /></a>
+                            </div>
+                            <h3>Movie name</h3>
+                        </div>
+                    </div>
+                    <div className='carousel__item'>
+                        <div className='carousel__img-container'>
+                            <img src={poster_3} />
+                            <div className='carousel__ticket'>
+                                <a href="#"><Ticket /></a>
+                            </div>
+                            <h3>Movie name</h3>
+                        </div>
+                    </div>
+                </div>
+
+            </div>
+            <div className='carousel__arrow' onClick={handleRightArrow}>
+                <img src={rightArrow} />
+            </div>
+        </div>
+    )
+}
```

Comment:
Add an empty line. 
There is the same mistake in soem other files, so fix it there as well.

Reply:
fixed